includes/attendee/attendee-repository.php

```diff
@@ -128,4 +128,46 @@ function ( object $row ) {
 			$rows
 		);
 	}
+
+	/**
+	 * Add a user as host of an event.
+	 *
+	 * @param Attendee $attendee The attendee to add as host.
+	 * @return void
+	 */
+	public function add_host( Attendee $attendee ): void {
+		$this->update_host( $attendee, 1 );
+	}
+
+	/**
+	 * Remove a user as host of an event.
+	 *
+	 * @param Attendee $attendee The attendee to remove as host.
+	 * @return void
+	 */
+	public function remove_host( Attendee $attendee ): void {
```

Comment:
Similarly to my other comment about `add_host()`, I think we could use the existing `remove_attendee()` method instead of adding this one.

Reply:
The `remove_attendee()`removes the user from attending to the event, and what I want to do is to remove her as host, maintaining her assistance to the event.